grouper/api/main.py

```diff
@@ -3,8 +3,10 @@
 import sys
 from typing import TYPE_CHECKING
 
+from expvar.stats import stats
```

Comment:
or we could be even more opaque and do `from expvar.stats.stats import set_label` jk